framework/auth/core.py

```diff
@@ -1228,6 +1227,11 @@ def merge_user(self, user):
             user_settings.merge(addon)
             user_settings.save()
 
+        # Disconnect signal to prevent emails being sent about being a new contributor when merging users
+        # be sure to reconnect it at the end of this code block.
+        from website.project.signals import contributor_added
+        from website.project.views.contributor import notify_added_contributor
```

Comment:
That will lead to an import error due to circular import. I will add a comment saying so.